Sources/BraintreePayPalNativeCheckout/BTPayPalNativeRequest.swift

```diff
@@ -6,17 +6,164 @@ import BraintreeCore
 import BraintreePayPal
 #endif
 
-protocol BTPayPalNativeRequest {
-    var hermesPath: String { get }
-    var paymentType: BTPayPalPaymentType { get }
+@objcMembers public class BTPayPalNativeRequest: NSObject {
 
-    func parameters(with configuration: BTConfiguration) -> [AnyHashable: Any]
-}
+    // MARK: - Public Properties
+    // next_major_version: subclass BTPayPalRequest once BraintreePayPal is in Swift.
```

Comment:
Can we make this note more clear that this class has duplicate logic of `BTPayPalRequest` and we should eventually remove the duplication?